packages/core/src/MPEContract.js

```diff
@@ -32,7 +32,7 @@ class MPEContract {
    * @type {string}
    */
   get address() {
-    return this.contract.address;
+    return this._contract.options.address;
```

Comment:
What is need to have separately for options?
I like the previous approach of contract.address.

Reply:
the `contract.address` is being undefined at times. 
The new property `contract.options.address` is being more stable. 
It is part of the latest web3 https://web3js.readthedocs.io/en/v1.3.0/web3-eth-contract.html#options-address